Tests/NIOTests/AcceptBackoffHandlerTest.swift

```diff
@@ -96,7 +96,9 @@ public class AcceptBackoffHandlerTest: XCTestCase {
             return readCountHandler.readCount
         }.wait())
 
-        XCTAssertTrue(try serverChannel.pipeline.remove(name: acceptHandlerName).wait())
+        try serverChannel.pipeline.remove(name: acceptHandlerName).mapIfError { _ in
+            XCTFail("Handler was not removed")
+        }.wait()
```

Comment:
This conversion can just be `XCTAssertNoThrow(try serverChannel.pipeline.remove(name: acceptHandlerName).wait())`.

Reply:
Fixed :)